app/Http/Controllers/PollerController.php

```diff
@@ -5,26 +5,18 @@
 use App\Models\Device;
 use App\Models\Poller;
 use App\Models\PollerCluster;
-use App\Models\PollerGroups;
+use App\Models\PollerGroup;
+use Carbon\Carbon;
 use Illuminate\Http\Request;
+use LibreNMS\Config;
 
 class PollerController extends Controller
 {
     public $rrdstep;
-    public $defaultPollerId;
-
-    public $defaultGroup = [
-        'id' => 0,
-        'group_name' => 'General',
-        'descr' => ''
-    ];
-    public $defaultPollerMarker = '(default Poller)';
 
     public function __construct()
     {
-        $this->authorizeResource(PollerGroups::class, 'poller_groups'); // FIXME is this correct? not a resource anymore
         $this->rrdstep = \LibreNMS\Config::get('rrd.step');
```

Comment:
could also be changed to `Config::get('rrd.step');` :)

Reply:
Oops, It was, but then I changed things around and it got reverted :)